sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -75,6 +74,9 @@ def test_dict_learning_overcomplete():
 def test_dict_learning_positivity(transform_algorithm,
                                   positive_code,
                                   positive_dict):
+    if transform_algorithm == 'lars' and (positive_code or positive_dict):
+        raise SkipTest()
```

Comment:
I'm not sure if skip is appropriate for excluding some combination of parameters. Might be good to work out what pytest best practice is